package.json

```diff
@@ -1128,6 +1128,16 @@
           "default": 1,
           "markdownDescription": "Scaling of Hover Preview."
         },
+        "latex-workshop.hover.preview.newcommand.parseTeXFile.enabled": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable newcommands defined in the current TeX file to be included in Hover Preview."
+        },
+        "latex-workshop.hover.preview.newcommand.newcommandFile": {
+          "type": "string",
+          "default": "",
+          "markdownDescription": "Set the path of a file in which newcommands of LaTeX defined for Hover Preview. If the path is relative, it is joined with the root dir."
```

Comment:
I guess there is a typo in the description. Can you fix this?

Reply:
I cannot find. Please tell me.

Reply:
What about: "Set the path of a file containing newcommands to be used in Hover Preview. If the path is relative, it is joined with the root dir."

Reply:
Thanks.